omx_components/src/mfx_omx_vdec_component.cpp

```diff
@@ -617,7 +617,11 @@ OMX_ERRORTYPE MfxOmxVdecComponent::DealWithBuffer(
                 m_pSurfaces->SetMaxErrorCount(port_info->nCurrentBuffersCount+1);
             }
         }
-        else mfx_sts = MFX_ERR_MEMORY_ALLOC;
+        else 
+	{
+	    MFX_OMX_LOG_ERROR(g_OmxLogLevel, "Failed to allocate buffers, maybe all buffers are allocated!");
```

Comment:
Is it better to add the information of these buffers (pBufferHeader, pBuffer)? For example,
MFX_OMX_LOG_ERROR(g_OmxLogLevel, "Failed to allocate the buffer, pBufferHeader=%p, pBuffer=%p", pBufferHeader, pBuffer);